recipes/pyLBL/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "pyLBL" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/GRIPS-code/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 010e8b395e8ff6f05c551727e579a7852a7e3808a1c2be47797fa3c2335efbe2
+
+build:
+  skip: true  # [py<36]
```

Comment:
Conda-forge is already only building on Python >=3.7.
```suggestion
```

Reply:
Fixed, thank you.